src/main/java/com/team/planopedia/API/translators/YelpAPI.java

```diff
@@ -0,0 +1,96 @@
+package com.team.planopedia.API.translators;
+
+
+/**
+ * This Class is meant to call the Yelp API given data from YelpAdapter
+ * Last updated August 31, 2021
+ * @Author Nimra Sami
+ * @Author Arturo Serdan
+ */
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.json.*;
+
+public class YelpAPI {
+    private static final String baseUrlYelp = "https://api.yelp.com";
+    private static final String callActionYelp = "/v3/businesses/search?";
+    public static int numberOfValuesForEachRestaurant = 5;
+
+    /**
+     * Method that makes a API call to the Yelp API for a list of restaurants in
+     * the location given by the user
+     *
+     * @param _term breakfast, lunch, or dinner, etc.
+     * @param _city city where user is located
+     * @param _limit search limit chosen by the user
+     * @return array values
+     */
+    //@Override
+    public String[] getRestuarants(String _term, String _city, int _limit) {
```

Comment:
This should be returning a Map object (key value pairs) instead of an array of strings.